_layouts/studentDetails.html

```diff
@@ -18,40 +18,53 @@
 name_with_initials: K. A. I. Fernando
 preferred_short_name: Ishan
 preferred_long_name: Ishan Fernando
-honorific: Ishan Fernando
+honorific: Mr.
 
 email_faculty: e18098@eng.pdn.ac.lk
 email_personal: personalMail@gmail.com
 
 location: Negombo
 
-url_cv: N/A
-url_website: N/A
-url_linkedin: N/A
+url_cv: #
+url_website: #
+url_linkedin: #
 url_github: https://github.com/ishanfdo18098
 url_facebook: https://facebook.com/ishanfdo
-url_researchgate: N/A
-url_twitter: N/A
+url_researchgate: #
+url_twitter: #
 
-projects_done: N/A
+projects_done: []
 image_url: "images/students/e18/e18098.jpg"
 ---END---
 
 NOTES:
-1) if url = N/A it will show N/A, for everything else it will show the View button
+1) if url = # it will show N/A, for everything else it will show the View button
+2) Social Media links, location are optional
 
 {%- endcomment -%}
 
+
+
 <!-- Current path START-->
 <main class="container">
     <div class="bg-secondary p-3 pb-1 rounded">
         <!-- <h1 class="text-white">Student Details</h1> -->
         <nav aria-label="breadcrumb">
             <ol class="breadcrumb">
-                <li class="breadcrumb-item"><a href="../../../" class="link-light">Home</a></li>
+                <li class="breadcrumb-item"><a href="{../../../}" class="link-light">Home</a></li>
```

Comment:
This link is not working on Student Details Page. Btw why does it have curly braces?